activitysim/abm/models/util/tour_od.py

```diff
@@ -0,0 +1,980 @@
+# ActivitySim
```

Comment:
Is there a reason this code is separated from activitysim.abm.models.tour_od_choice? Do other models use these utility functions?

Reply:
Yeah. From the initial PR comment:

> WHY: Much like the location_choice.py models ("workplace_location_choice" and "school_location_choice") rely on the tour_destination.py submodule to do their heavy lifting, tour_od.py helps to keep the "tour_od_choice" model step definition code to a minimum.

More than anything I just thought I was conforming to an ActivitySim design pattern. Could just as easily be combined IMO.